ddsrouter_test/compose/test_cases/stress/multi_routers_cloud/compose.yml

```diff
@@ -0,0 +1,280 @@
+#  │ │ ...           │   │ │ │    │ │    │ │     └───────────────┘ │
+#  │ │ publisher_8   ├───┘ │ │    │ │    │ │                       │
+#  │ │ (edge_1_net)  │     │ │    │ │    │ │     ┌───────────────┐ │
+#  │ └───────────────┘     │ │    │ │    │ │     │ subscriber_3  │ │
+#  │                       │ │    │ │    │ └─────► (edge_2_net)  │ │
+#  │ ┌───────────────┐     │ │    │ │    │       └───────────────┘ │
+#  │ │ publisher_9   │     │ │    │ │    │                         │
+#  │ │ ...           │     │ │    │ │    │   ┌───────────────────┐ │
+#  │ │ publisher_12  ├─────┘ │    │ │    │   │subscriber_reliable│ │
+#  │ │ (edge_1_net)  │       │    │ │    └───► (edge_2_net)      │ │
+#  │ └───────────────┘       │    │ │        └───────────────────┘ │
+#  │                         │    │ │                              │
+#  │ ┌──────────────────┐    │    │ │                              │
+#  │ │publisher_reliable│    │    │ │                              │
+#  │ │ (edge_1_net)     ├────┘    │ │                              │
+#  │ └──────────────────┘         │ │                              │
+#  └──────────────────────────────┘ └──────────────────────────────┘
+
+
+
+
+
+
+
+services:
+
+  # ROUTERS
+  ddsrouter_cloud_1:
+    image: ${DDSROUTER_COMPOSE_TEST_DOCKER_IMAGE}
+    container_name: ddsrouter_cloud_1
+    networks:
+      - cloud_edge_net
+    volumes:
+      - ./ddsrouter_cloud_1.yaml:/config.yaml
+    command: ddsrouter -c /config.yaml --timeout 15
+
+  ddsrouter_cloud_2:
+    image: ${DDSROUTER_COMPOSE_TEST_DOCKER_IMAGE}
+    container_name: ddsrouter_cloud_2
+    depends_on:
+      - ddsrouter_cloud_1
+    networks:
+      - cloud_edge_net
+    volumes:
+      - ./ddsrouter_cloud_2.yaml:/config.yaml
+    command: ddsrouter -c /config.yaml --timeout 15
+
+  ddsrouter_1:
+    image: ${DDSROUTER_COMPOSE_TEST_DOCKER_IMAGE}
+    container_name: ddsrouter_1
+    depends_on:
+      - ddsrouter_cloud_1
+    networks:
+      - cloud_edge_net
+      - edge_1_net
+    volumes:
+      - ./ddsrouter1.yaml:/config.yaml
+    command: ddsrouter -c /config.yaml --timeout 15
+
+  ddsrouter_2:
+    image: ${DDSROUTER_COMPOSE_TEST_DOCKER_IMAGE}
+    container_name: ddsrouter_2
+    depends_on:
+      - ddsrouter_cloud_2
+    networks:
+      - cloud_edge_net
+      - edge_2_net
+    volumes:
+      - ./ddsrouter2.yaml:/config.yaml
+    command: ddsrouter -c /config.yaml --timeout 15
+
+ # SUBSCRIBERS
+  subscriber_reliable:
+    image: ${DDSROUTER_COMPOSE_TEST_DOCKER_IMAGE}
+    container_name: subscriber_reliable
+    depends_on:
+      - ddsrouter_2
+    networks:
+      - edge_2_net
+    volumes:
+      - ../../../scripts:/scripts
+    command: python3 /scripts/execute_and_validate_subscriber.py --transient --exe install/AdvancedConfigurationExample/examples/cpp/dds/AdvancedConfigurationExample/AdvancedConfigurationExample --samples 10 --timeout 15 --args "--samples 10 --domain 105 --topic LargeDataTopic_reliable --reliable"
```

Comment:
Why is this `transient`?

Reply:
it's a copy and paste error

Reply:
I believe you forgot to remove it from the `simple/reliable` and from `repeater/reliable`.